packages/@lwc/engine-dom/src/index.ts

```diff
@@ -5,15 +5,12 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 
+// Globals -----------------------------------------------------------------------------------------
+import '@lwc/features';
+
```

Comment:
This explicit import is needed in `@lwc/engine-dom`, `@lwc/engine-server`, and `@lwc/synthetic-shadow`, but not in `@lwc/engine-core` because that package already exports `setFeatureFlag*` from `@lwc/features`.